package.json

```diff
@@ -9,6 +9,8 @@
     "lodash": "^4.17.20",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-native": "^0.63.4",
+    "react-native-web": "^0.14.10",
```

Comment:
React Native is for Android IOS app development. What's it's usage here?

Reply:
For the Linking and BackHandler functions used in the intro component.
If they are not needed and can be done in another way, I'll remove them.

Reply:
If its about handling the back button there should probably be away through react-router,